src/chain/store/chain_store.rs

```diff
@@ -485,17 +485,25 @@ where
 }
 
 /// Returns parent message receipt given `block_header` and message index.
-pub fn get_parent_reciept<DB>(
+pub fn get_parent_receipt<DB>(
     db: &DB,
     block_header: &BlockHeader,
     i: usize,
 ) -> Result<Option<Receipt>, Error>
 where
     DB: Blockstore,
 {
+    // Try Receipt_v2 first
```

Comment:
I'm not sure the logic for containing this should be in the `chain_store`. Seems more like a `shim` thingy.

Reply:
Fixed